src/test/java/teammates/test/cases/action/InstructorCourseRemindActionTest.java

```diff
@@ -171,39 +171,31 @@ public void testExecuteAndPostProcess() throws Exception {
                 Const.ParamsNames.INSTRUCTOR_EMAIL, invalidEmail
         };
         
-        try {
-            remindAction = getAction(addUserIdToParams(instructorId, submissionParams));
-            redirectResult = (RedirectResult) remindAction.executeAndPostProcess();
-            signalFailureToDetectException();
-        } catch (EntityNotFoundException e) {
-            ignoreExpectedException();
-        }
+        testExecuteAndPostProcess_assertEntityNotFoundException(instructorId, submissionParams);
         
         submissionParams = new String[]{
                 Const.ParamsNames.COURSE_ID, courseId,
                 Const.ParamsNames.STUDENT_EMAIL, invalidEmail
         };
         
-        try {
-            remindAction = getAction(addUserIdToParams(instructorId, submissionParams));
-            redirectResult = (RedirectResult) remindAction.executeAndPostProcess();
-            signalFailureToDetectException();
-        } catch (EntityNotFoundException e) {
-            ignoreExpectedException();
-        }
+        testExecuteAndPostProcess_assertEntityNotFoundException(instructorId, submissionParams);
         
         submissionParams = new String[]{
                 Const.ParamsNames.COURSE_ID, "invalidCourseId"
```

Comment:
According to the test case design principle, two invalid inputs should not appear at the same time for one test case. I suggest to change this test case.